### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by juandopazo** [stjohnjohnson/jenkins-mocha#2]
So the problem with Windows support is really simple: `node_modules/.bin/_mocha` doesn´t work on Windows. The one that works in all platforms is `node_modules/mocha/bin/_mocha`.

So instead of...

``` js
shell.exec(
        path.join(directories.base, 'istanbul') +
        ' cover --dir ' + directories.coverage + ' -- ' +
        path.join(directories.base, '_mocha') +
        ' --reporter spec-xunit-file ' + args.join(' ')
    )
```

You'd just need to do...

``` js
shell.exec(
        path.join(directories.base, 'istanbul') +
        ' cover --dir ' + directories.coverage + ' -- ' +
        path.join('node_modules', 'mocha', 'bin', '_mocha') +
        ' --reporter spec-xunit-file ' + args.join(' ')
    )
```

Would you be up for changing it?

**Comment by stjohnjohnson** [comment]
That's so strange that Mocha's .bin doesn't work.  But this seems reasonable.

**Comment by stjohnjohnson** [comment]
Available in `jenkins-mocha@2.0.1`
